docs/requirements.txt

```diff
@@ -1,10 +1,16 @@
 # ReadTheDocs uses the `environment.yaml` so make sure to update that as well
 # if you change this file
--r ../requirements.txt
```

Comment:
For my education: why can we no longer reference the requirements file from the parent directory?

Reply:
We can still reference it, but we don't need all the requirements of the project for the docs, so I removed it to speed up a bit the docs build.

Reply:
Ah ok! That sounds like a good thing to do :)

Reply:
I think we should still get all the requirements from the package to ensure that autodoc populates correctly. This also helps reduce maintenance burden because ~*every* time we add a real dependency to the package the docs builds break until we add a duplicate to docs/requirements.txt. This has happened several times across projects before adding the package requirements to the build.

Reply:
This makes sense, @minrk, I'll put back the package requirements.